### DISCUSSION TITLE
Is hide_end_of_buffer a good name?

### DISCUSSION THREAD

**Discussion by bryanboateng** [projekt0n/github-nvim-theme#206] [General]
Hi,

I set up the theme and was wondering why I could not see certain listchars (eol, precedes, extends). After searching for a long time I found `hide_end_of_buffer`. I find the name and description a bit unintuitive because the setting also controls the afformentioned listchars. Would it make sense to improve the naming and/or the description?

**Comment by tmillr** [comment]
Thanks for brining this up. This is a good point, setting this hl group might have side-effects. Perhaps we should investigate this. See #346
